src/doc/unstable-book/src/macro-vis-matcher.md

```diff
@@ -0,0 +1,14 @@
+# `macro_vis_matcher`
+
+The tracking issue for this feature is: [#41022]
+
+With this feature gate enabled, the [list of fragment specifiers][frags] gains one more entry:
+
+* `vis`: a visibility qualifier. Examples: nothing (default visibility); `pub`; `pub(crate)`.
+
+A `vis` variable may be followed by a comma, ident, type, or path.
```

Comment:
It seems harmless to me. I can't imagine using `,` as an "operator" in a visibility specifier (at last outside of parens or something).